presto-main/src/main/java/com/facebook/presto/sql/gen/InCodeGenerator.java

```diff
@@ -41,10 +43,45 @@
 import static com.facebook.presto.sql.gen.ByteCodeUtils.ifWasNullPopAndGoto;
 import static com.facebook.presto.sql.gen.ByteCodeUtils.invoke;
 import static com.facebook.presto.sql.gen.ByteCodeUtils.loadConstant;
+import static com.google.common.base.Verify.verify;
 
 public class InCodeGenerator
         implements ByteCodeGenerator
 {
+    enum SwitchGenerationCase
+    {
+        DIRECT_SWITCH,
+        HASH_SWITCH,
+        SET_CONTAINS,
```

Comment:
no comma after last element. use ';' or nothing to match the rest of the code style